OSD-15767 : a fixed footer has been added below the terminal bar

Dee-6777 opened the pull request:
### What type of PR is this?

_feature_

### What this PR does / Why we need it?

This PR adds a fixed footer in the tabs section that lists out all the shortcuts are available for the user to interact with.

### Which Jira/Github issue(s) does this PR fix?

_Resolves_ #[OSD-15767](https://issues.redhat.com/browse/OSD-15767)

### Pre-checks (if applicable)

- [X] Ran unit tests locally against the changes
- [ ] Included documentation changes with PR

devppratik commented:
@Dee-6777 please change the pagebar background color similar to TMUX as mentioned in the description

Dee-6777 commented:
> @Dee-6777 please change the pagebar background color similar to TMUX as mentioned in the description
Added background color similar to TMUX and added more shortcuts.
![Screenshot from 2023-04-03 17-49-21](https://user-images.githubusercontent.com/73513838/229507636-f61f5c83-5d7e-49ed-a9d5-e05e47c03b2d.png)

supreeth7 commented:
/lgtm
